src/pages/en/guides/deploy/github.md

```diff
@@ -25,45 +25,52 @@ You can deploy an Astro site to GitHub Pages by using [GitHub Actions](https://g
     name: Github Pages Astro CI
 
     on:
-      # Trigger the workflow every time you push to the `main` branch
-      # Using a different branch name? Replace `main` with your branch’s name
+     # Trigger the workflow every time you push to the `main` branch
```

Comment:
Do you think it would be helpful to include this notice we used to have for folks have have different branch names?

```suggestion
      # Trigger the workflow every time you push to the `main` branch
      # Using a different branch name? Replace `main` with your branch’s name
```

Reply:
@delucis @blackgirlbytes If this instruction is still correct (replace `main` if your'e using a different branch), then I'd be in favour of keeping it in. I would say, only remove if it's wrong or misleading.